src/Commands/Site/InfoCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Site;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Collections\Sites;
+use Terminus\Models\Environment;
+
+/**
+ * Class InfoCommand
+ * @package Pantheon\Terminus\Commands\Site
+ */
```

Comment:
My editor suggested and generated this PHPDoc. I'm fine with removing it for now if it lints correctly or making a decision and applying that now. Personally I don't know where the utility of them is derived so... /shrug.